app.js

```diff
@@ -21,6 +24,21 @@ app.get('/js/main.js', (req, res) => {
   });
 });
 
+app.get('/js/login.js', (req, res) => {
```

Comment:
Manually replacing config variables as client-side JS is served is a common practice. Though they often use tools like https://github.com/yahoo/express-state to do it. Perhaps I can add this in a future release.